ui/components/multichain/create-named-snap-account/create-named-snap-account.test.tsx

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable jest/require-top-level-describe */
+import React from 'react';
+import { InternalAccount } from '@metamask/keyring-api';
+import { fireEvent, renderWithProvider, waitFor } from '../../../../test/jest';
+import configureStore from '../../../store/store';
+import mockState from '../../../../test/data/mock-state.json';
+import { CreateNamedSnapAccountProps } from './create-named-snap-account';
+import { CreateNamedSnapAccount } from '.';
+
+const render = (
+  props: CreateNamedSnapAccountProps = {
+    onActionComplete: async () => Promise.resolve(),
+    address: '0x2a4d4b667D5f12C3F9Bf8F14a7B9f8D8d9b8c8fA',
+    snapSuggestedAccountName: 'Suggested Account Name',
+  },
+) => {
+  const store = configureStore(mockState);
+  return renderWithProvider(<CreateNamedSnapAccount {...props} />, store);
+};
+
+const mockSetAccountLabel = jest.fn().mockReturnValue({ type: 'TYPE' });
+const mockGetNextAvailableAccountName = jest
+  .fn()
+  .mockReturnValue('Snap Account 2');
+
+jest.mock('../../../store/actions', () => ({
+  setAccountLabel: (...args: string[]) => mockSetAccountLabel(...args),
+  getNextAvailableAccountName: (...args: InternalAccount[]) =>
+    mockGetNextAvailableAccountName(...args),
+}));
+
+describe('CreateNamedSnapAccount', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('displays account name input and suggested name', async () => {
```

Comment:
Don't we miss a test when `snapSuggestedAccountName` is undefined, so that we fallback to the logic of the next `Snap Account <index>`?

This comment goes in pair with this one in `ui/components/multichain/create-named-snap-account/create-named-snap-account.tsx` file.

Reply:
WKIP [fb49290](https://github.com/MetaMask/metamask-extension/pull/25191/commits/fb492909f93eb86d2e44f304da8c5a08f26e9322)